cluster/ubuntu-cluster/configure.sh

```diff
@@ -21,6 +21,10 @@
 
 set -e
 
+CONFIG_DIR=`dirname "$0"`
+CONFIG_DIR=`cd "$CONFIG_DIR"; pwd`
```

Comment:
Em ... I'm not against that, just keep it. Would u please add comments to your change and then squash to one commit? And then we can ship it.